data/human/human missions.txt

```diff
@@ -3601,3 +3601,107 @@ mission "Mafia Extortion"
+			choice
+				`	"I'd like to give it a try." (Bet 400 credits).`
+					goto firstgameplay
+				`	(Wait and watch.)`
+					goto firstgamewatch
+				`	(Walk away.)`
+					decline
+			label firstgamewatch
+			`	A short woman volunteers. The game progresses in a similar manner; by the end, you are certain the ball is underneath the center cup. Sure enough, the woman chooses that cup, and she walks away 400 credits richer.`
+			`	"The next buy in is a larger prize!" the announcer calls after her before turing to address the crowd. "Six thousand credits, only for the proudest of players." The magician shakes her head softly, frowning at the cups.` 
+			choice
+				`	"I'll do it!" (Bet 6000 credits).`
+					goto secondgameplay
+				`	(Continue watching.)`
+					goto secondgamewatch
+				`	(Walk away.)`
+					decline
+			label firstgameplay
+			`	You pay 400 credits and sit across from the magician. "Double or nothing, bet your bet!" the announcer repeats. "Are you a player or a puppet?"`
+			`	The magician places the ball beneath the center and begins moving the cups. This one seems faster, but that might just be from the pressure. You keep your eyes trained on the cup as it shifts back and forth, around in circles, and occasionally remains untouched. By the end, you're sure it's in the center cup.`
+			`	"What's your choice?" the magician asks, smirking.`
+			choice
+				`	"Left."`
+					goto wrong
+				`	"Center."`
+					goto correct
+				`	"Right."`
+					goto wrong
+			label wrong
+			`	"Better luck next time," she says as she reveals your incorrect choice. The ball was in the center; you feel silly for not going with your gut.`
+			`	"The next buy-in is a larger prize!" the announcer calls. "Six thousand credits, reserved for the most daring. As previous player, you have first priority."`
+			action
+				payment -400
+			choice
+				`	"Let's go again."`
+					goto secondgameplay
+				`	"I think I'm done."`
+					goto secondgamewatch
+			label correct
+			`	She asks if you're sure, and you stick with your gut. With a grimace, she lifts the cup to reveal the ball and counts out your winnings. "And the player prevails!" the announcer says. "Earning a small bounty. But wait: the next buy-in is bigger yet! Six thousand credits, reserved for the most daring. As previous player, you have first priority."`
+			action
+				payment 400
+			choice
+				`	"Let's go again."`
+					goto secondgameplay
+				`	"I think I'm done."`
+					goto secondgamewatch
+			label secondgameplay
+			`	You place your 6000 unit bet on the table, and the game begins. The pattern is slightly longer, but once the magician's hands stop, you are certain the ball lies beneath the left cup.`
```

Comment:
```suggestion
			`	You place your 6000 credit bet on the table, and the game begins. The pattern is slightly longer, but once the magician's hands stop, you feel like the ball lies beneath the left cup.`
```
Not sure I've seen credits called units before. Let me know if you know of an in-game example of it!